collections/_contributors/keenal-shah.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Hey @Keenal this is something which is earned with time and we as a community keep reviewing the contributions made and with time the profile is added to the members list. Not in this way.
Please remove this file and the image too.

Reply:
Oh, sorry, I wasn't aware about it. I'll remove it.

Reply:
@Nikhil-Ladha I removed it. Let me know if anything else is to be done from my end for this PR.

Reply:
This PR is a great start, though, @keenal.  @Nikhil-Ladha, I, and other community managers would love to have your profile on the site. Join the community's Slack and let's get you some resources so that you can dig in even further.

Reply:
Perfect, I'll join you guys there, thanks for the invite!